apps/web/src/lib/mail/sendgrid.ts

```diff
@@ -0,0 +1,23 @@
+import sgMail from "@sendgrid/mail";
+
+async function SendMail(to: string, subject: string, html: string) {
```

Comment:
Funktionen werden anders als in Go etc. hier in camelCase geschrieben. Ist kein muss aber das haben wir bisher immer so gemacht.